salt/modules/mount.py

```diff
@@ -432,6 +434,141 @@ def match(self, line):
         return True
 
 
+class _filesystems_entry(object):
+    '''
+    Utility class for manipulating filesystem entries. Primarily we're parsing,
+    formatting, and comparing lines. Parsing emits dicts expected from
+    fstab() or raises a ValueError.
+
+    Note: We'll probably want to use os.normpath and os.normcase on 'name'
+    '''
+
+    class ParseError(ValueError):
+        '''
+        Error raised when a line isn't parsible as an fstab entry
+        '''
```

Comment:
I'm not sure why a separate exception class is needed here, specifically one that is private to this class.